src/binding.cc

```diff
@@ -22,7 +22,7 @@ static_assert(sizeof(two_pointers_t) ==
                   sizeof(std::shared_ptr<v8::BackingStore>),
               "std::shared_ptr<v8::BackingStore> size mismatch");
 
-static_assert(sizeof(v8::ScriptOrigin) == sizeof(size_t) * 7,
+static_assert(sizeof(v8::ScriptOrigin) <= sizeof(size_t) * 8,
```

Comment:
This over-allocates slightly on 64 bits architectures but I think that's okay, better than #ifdefs.

Reply:
Yeah, cool with me.